src/src/CustomBuild/define_plugin_sets.h

```diff
@@ -3636,6 +3636,12 @@ To create/register a plugin, you have to :
     #endif
   #endif
 
+  #if defined(FEATURE_THINGSPEAK_EVENT) || defined(FEATURE_OPENMETEO_EVENT)
```

Comment:
These should check for the defines to be true/1, when 0 they are also defined...
`#if FEATURE_THINGSPEAK_EVENT || FEATURE_OPENMETEO_EVENT`

Reply:
Ahh true...

Reply:
> Ahh true...

... or false ;)